components/server/src/github/github-auth-provider.ts

```diff
@@ -82,26 +82,33 @@ export class GitHubAuthProvider extends GenericAuthProvider {
         }
 
         const fetchCurrentOrg = async () => {
-            if(!org) return {state:"active"};
-            const response = await api.orgs.getMembershipForAuthenticatedUser({
-                org,
-            });
-            if (response.status !== 200) {
-                throw new GitHubApiError(response);
-            }
-            const {state} = response.data;
-            if(state != "active"){
-                throw new GitHubApiError(response);
+            if (!org) return {state:"active"};
```

Comment:
Test with blank org.  If the return is string, this is a bug i introduced . 
if (!org) return {state:"active"}; => if (!org) return "active";

Reply:
Tested, no issues